### DISCUSSION TITLE
Add a time duration for every section of the course in the updraft.cyfrin.com

### DISCUSSION THREAD

**Discussion by tahamasood150** [Cyfrin/security-and-auditing-full-course-s23#10] [General]
### Suggestion about visibility
Consider adding a time duration to every section of the course will help the viewers to actually analyze how much time the section can gonna take to be able to complete.

Thanks.

**Comment by cromewar** [comment]
Thanks for the suggestion @tahamasood150

**Reply by cromewar** [reply]
@tahamasood150 we already provide the time duration on the lesson description:
![image](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/20199180/e4bacaab-129e-4154-9019-7eaa5796e1a2)

Any other idea in mind?

**Reply by tahamasood150** [reply]
Hi, I mean for each section Like:

Section1: Review 
  Duration: x mins

**Reply by PatrickAlphaC** [reply]
smart, will add
